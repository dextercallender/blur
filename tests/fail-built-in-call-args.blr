void main() {
	print("hi", "there");
}