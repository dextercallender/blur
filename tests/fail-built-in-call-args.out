Fatal error: exception Failure("Incorrect number of args for function call print. Expecting 1 args but got 2")
